pkgs/development/python-modules/effdet/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+# build inputs
+, torch
+, torchvision
+, timm
+, pycocotools
+, omegaconf
+# check inputs
+, pytestCheckHook
+}:
+let
+  pname = "effdet";
+  version = "0.4.1";
+in
+buildPythonPackage {
+  inherit pname version;
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-rFWJ/TBKVlDCAZhrLvX44QwREJOnGxxJ+muIF3EIErU=";
+  };
+
+  propagatedBuildInputs = [
+    torch
+    torchvision
+    timm
+    pycocotools
+    omegaconf
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
```

Comment:
The package has no tests and `pytestCheckHook` fails the build when pytest discovers no tests. Think we should add `doCheck = false;` here with a comment.